server.go

```diff
@@ -176,23 +171,20 @@ func (s *Server) Boot() error {
 func (s *Server) goAcceptConnection(listener net.Listener) {
 	s.wait.Add(1)
 	go func(listener net.Listener) {
-	loop:
```

Comment:
just a tiny refactor to make it easier to reason about (break LABEL is a little confusing to me, I assume it is for others too).